src/components/Confirm/Confirm.js

```diff
@@ -47,6 +47,8 @@ class Confirm extends Component {
         ? 'NO_FACE_ERROR'
         : 'MULTIPLE_FACES_ERROR'
     }
+    // return a generic error if the status is 422 and the key is none of the above
+    return 'SERVER_ERROR'
```

Comment:
shouldn't we name this `UNKNOWN_ERROR`?

Because it could be a legitimiate error like bad document type, or bad filetype.
Naming it `SERVER_ERROR` makes it seem we know what happened (it was the SERVER that had an error)

Reply:
shouldn't we also unit test this?

That will make sure we always have this catch all option.

Reply:
I agree, this key should be renamed as `GENERIC_ERROR` or `UPLOAD_ERROR` as we return it every time that there is a generic error that happens during uploads. We already use a similar key called `GENERIC_CLIENT_ERROR` for errors that happen on cross device, although in this case, it might be better to use `UPLOAD_ERROR`.

Reply:
After discussing it with the team, we will rename this key as `REQUEST_ERROR`